index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="./assets/css/style.css" />
+        <title>Elizabeth's Portfolio Page</title>
+    </head>
+  <body>
+      <!--navigation bar of the web page-->
+      <header class="header">
+      <h1>Elizabeth's Portfolio</h1>
+      <nav class=nav-container> 
+      <div>
+        <button type="email"> <img src="images/designes/EmailIcon.jpg" class="email"> Email</button>
+        <a href="Name"> 
+      </div>
+      <div>
+        <button type="GitHub"> <img src="images/designes/GitHubIcon.gif" class="github">Git Hub</button>
+        <a href="GitHub"> 
+      </div>
+      <div>
+        <button type="linkedIn"> <img src="images/designes/LinkedInIcon.png" class="linkedin"> LinkedIn</button>
+        <a href="Link 3"> 
+      </div>
+      <!--dive that consist a little logo image -->
+      <div>
+          <div class="logo-box">
+          <a href="Logo">
```

Comment:
Does this `img` need to be wrapped in a `<a>`? It doesn't seem to link to anywhere.